### ISSUE TITLE
Spatial/Node2D locks should not be saved through metadata

### ISSUE THREAD

**Issue by kubecz3k** [godotengine/godot#13098]
**Operating system or device, Godot version, GPU Model and driver (if graphics related):**
450bdda97a933ed9081efa20bddc71ee0b8286c9

**Issue description:**
According to discussion in #12359 it seems locks should not be saved inside scene metadata anymore. Instead there should be new bool property inside node2d/spatial that should not be visible in inspector. As far as we discussed on IRC this still could be considered for 3.0

**Comment by groud** [comment]
I guess the "selectable children" metadata could be moved too :)

**Comment by akien-mga** [comment]
We have now entered release freeze for Godot 3.0 and want to focus only on release critical issues for that milestone. Therefore, we're moving this issue to the 3.1 milestone, though a fix may be made available for a 3.0.x maintenance release after it has been tested in the master branch during 3.1 development. If you consider that this issue is critical enough to warrant blocking the 3.0 release until fixed, please comment so that we can assess it more in-depth.

**Comment by KoBeWi** [comment]
I disagree. Metadata are perfect place to store editor-related info. They are all prefixed with `_edit_`, so it doesn't conflict with player's values.

**Comment by akien-mga** [comment]
Is this still relevant as of 4.0? I know we moved a lot of stuff out of scenes and to editor-specific folders, might include this.

**Comment by KoBeWi** [comment]
Some issues caused by metadata were fixed, so this is less relevant.
